DfE.FindInformationAcademiesTrusts/assets/sass/components/_side-navigation.scss

```diff
@@ -9,6 +9,10 @@
     @include govuk-responsive-margin(6, "top");
     @include govuk-responsive-margin(6, "bottom");
   }
+
+  .ds_side-navigation__expand {
+    @include govuk-font($size: 16, $weight: bold)
```

Comment:
On the [design system type](https://design-system.service.gov.uk/styles/type-scale/) scale `16` is equivalent to `govuk-body-s`. I think we should either use `24` (equivalent to `govuk-heading-m`, `govuk-body-l`) or `19` (equivalent to `govuk-heading-s`, `govuk-body`) - a designer might have stronger opinions!

<!--EndFragment-->
</body>
</html>